src/cswinrt/code_writers.h

```diff
@@ -6488,11 +6488,12 @@ bind<write_type_name>(type, typedef_name_type::CCW, true)
                 {
                     handler = (%) =>
                     {
-                        if (_state.del == null)
+                        % localDel = _state.del;
```

Comment:
or just 'var'

Reply:
Strange VS intellisense was complaining before when I used var, but its working now.  Will change it.